ci/scripts/wss.sh

```diff
@@ -51,8 +64,8 @@ wss.url=https://saas.whitesourcesoftware.com/agent
 ############
 checkPolicies=true
 forceCheckAllDependencies=false
-forceUpdate=false
-forceUpdate.failBuildOnPolicyViolation=false
+forceUpdate=true
+forceUpdate.failBuildOnPolicyViolation=true
```

Comment:
@sfc-gh-jfan are these good enough? any other parameter needs udpate?

Reply:
I pushed a new parameter to it, now it should be fine